test/SourceKit/CursorInfo/cursor_info_response.swift.response

```diff
@@ -0,0 +1,16 @@
+source.lang.swift.ref.module ()
+
+</LocalizationKey>
+OVERRIDES BEGIN
+OVERRIDES END
+RELATED BEGIN
+RELATED END
+TYPE INTERFACE BEGIN
+TYPE INTERFACE END
+MODULE GROUPS BEGIN
+MODULE GROUPS END
+ACTIONS BEGIN
+source.refactoring.kind.expand.switch.cases
+Expand Switch Cases
+(refactor)
```

Comment:
I wasn't sure how to test this as I couldn't find CursorInfo tests that diff the available actions, I'm open for tips!

Reply:
@nkcsgexi could you point @rockbruno in the right direction for testing this?

Reply:
@rockbruno could you try to clarify what are you trying to test? Are you trying to test printed lsp refactoring kind?

Reply:
Right @nkcsgexi, I was trying to confirm that the response is correctly attributing the new param for certain actions. My intention was to diff the raw response like when we added module refs: https://github.com/apple/swift/blob/master/test/SourceKit/Indexing/index_constructors.swift.response

Reply:
Perhaps directly checking the lines after `ACTIONS BEGIN` might a better way?

Reply:
You may need to change `sourcekitd-test` to print additionally the lsp refactoring kind name in here: https://github.com/apple/swift/blob/master/tools/SourceKit/tools/sourcekitd-test/sourcekitd-test.cpp#L1418 . After changing this, the printed action should be something like:
```
source.refactoring.kind.expand.switch.cases
Expand Switch Cases
lsp kind: Refactor
```

Reply:
Done! I saw the example later so it ended up being slightly different, but the test's purpose should be clearer now.